### PR TITLE
[Api/Tizen] Tizen internal API to construct pipeline

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Add Tizen internal API without checking the permission.

Signed-off-by: Jaeyun Jung <jy1210.jung@samsung.com>

### CODE REVIEW

File: api/capi/include/nnstreamer-tizen-internal.h [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ */
+/**
+ * @file nnstreamer-tizen-internal.h
+ * @date 02 October 2019
+ * @brief NNStreamer/Pipeline(main) C-API Header for Tizen Internal API. This header should be used only in Tizen.
+ * @see	https://github.com/nnsuite/nnstreamer
+ * @author MyungJoo Ham <myungjoo.ham@samsung.com>
```
Comment (myungjoo) [inline api/capi/include/nnstreamer-tizen-internal.h]: You are the author here :)

File: packaging/nnstreamer.spec [hunk 2]
```diff
@@ -195,6 +195,13 @@ Group:		Multimedia/Framework
 Requires:	capi-nnstreamer-single-new = %{version}-%{release}
 %description -n capi-nnstreamer-single-new-devel
 Developmental kit for Tizen Native new single-shot NNStreamer API.
+
+%package -n capi-nnstreamer-internal
+Summary:	Tizen internal API to construct the pipeline
+Group:		Multimedia/Framework
+Requires:	%{name} = %{version}-%{release}
+%description -n capi-nnstreamer-internal
+Tizen internal API to construct the pipeline without the permissions.
```
Comment (myungjoo) [inline packaging/nnstreamer.spec]: Having it named capi-* may confuse other Tizen developers (integrators especially...)

Let's name it nnstreamer-tizen-internal-capi-devel
And add ```Requires: capi-nnstreamer-devel = ...,```

File: api/capi/include/nnstreamer-tizen-internal.h (hunk 1 above)
Comment (jaeyun-jung) [reply api/capi/include/nnstreamer-tizen-internal.h]: PR updated to change pkg name

**Comment by myungjoo** [comment]
PTAL @again4you @helloahn @leemgs

**Comment by myungjoo** [comment]
Let's hope we can freeze the code for Tizen 5.5 M2 release :)

**Pull request merged**